README.md

```diff
@@ -48,6 +48,9 @@ import Component from 'vue-class-component'
   }
 })
 export default class App extends Vue {
+  // props are had to declare again
+  propMessage: string
```

Comment:
The example in this readme is not written in typescript. So we should not include types.